assets/stylesheets/_components.scss

```diff
@@ -107,7 +107,7 @@
 @import 'components/segmented-control/style';
 @import 'components/select-dropdown/style';
 @import 'components/seo/style';
-@import 'components/seo/settings-upgrade-nudge/style';
+@import 'my-sites/upgrade-nudge/expanded/style';
```

Comment:
This placement breaks our naming guidelines. If this is a completely separate component it should be namespaced (both folder name and class names) as `upgrade-nudge-expanded` or `expanded-nudge` and put into "blocks".